ASSEMBLY.md

```diff
@@ -54,8 +57,15 @@ The general rule is to fit the parts with the lowest Z-height first (i.e. the fl
 * The PSU module (U1301)
 * The Raspberry Pi Pico (if you want to solder it straight to the board) (U201)
 * Any 2.54mm pin headers and jumpers
-  * Don't worry about J906, J907 and J908 - they're just for debugging/testing/probing
-  * Ensure you leave out pin 8 on the J802 PC case audio connector, as most cases have a blank position on the mating connector to make sure you can't put it in backwards. Pin 8 is the top row, one in from the right; if you look carefully it's the only pad on that 10-way header that doesn't have a PCB trace running to it.
+  * Don't worry about J906, J907 and J908 - they're just for debugging/testing/probing. It's easier to put an oscilloscope probe directly into a hole.
+  * Ensure you leave out pin 8 on the J802 PC case audio connector, as most cases have a blank position on the mating connector to make sure you can't put it in backwards.<p>
+```
+.    +----+----+----+----+----+
+.    |  2 |  3 |  6 |  X | 10 |
+Case +----+----+----+----+----+
```

Comment:
What does the word *Case* mean or imply here?

Reply:
Case is printed on the PCB. I thought it might make the orientation more certain.

Reply:
Oh of course it is!